jobs/atc/spec

```diff
@@ -284,3 +284,11 @@ properties:
     description: |
       An optional prefix for emitted Riemann services
     default: ""
+
+  riemann.tags:
+    description: |
+      An optional list of tags in key: value format
```

Comment:
This should probably say 'map' or something rather than 'list'. List to me implies:

```
- env: dev
- foo: bar
```